operator/v1/types_network.go

```diff
@@ -80,6 +80,12 @@ type NetworkSpec struct {
 	// If not specified, sensible defaults will be chosen by OpenShift directly.
 	// Not consumed by all network providers - currently only openshift-sdn.
 	KubeProxyConfig *ProxyConfig `json:"kubeProxyConfig,omitempty"`
+
+	// exportNetworkFlows enables and configures the export of network flows from OVS
+	// by using protocols NetFlow, SFlow or IPFIX. Only supported on OVN-Kubernetes plugin.
```

Comment:
```suggestion
	// exportNetworkFlows enables and configures the export of network flow metadata from the pod network
	// by using protocols NetFlow, SFlow or IPFIX. Currently only supported on OVN-Kubernetes plugin.
```

I'd suggested these changes before but maybe you missed the fact that there was more than just capitalization.

1. "flow metadata" rather than "flows" because it's not actually exporting network traffic, it's just exporting information about network traffic
2. "the pod network" rather than "OVS" because flow export is not an OVS-specific feature and this could be supported by non-OVS-based plugins in the future
3. "Currently" only OVN-Kubernetes